<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QDialog" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QTextEdit" name="speechtextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="speechpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="votepushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>93</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>vote</string>
   </property>
  </widget>
  <widget class="QLabel" name="avatarlabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>aaaaaaaaaaaaaaaaaa</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerlabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player #</string>
   </property>
  </widget>
  <widget class="QLabel" name="votelabel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>225</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shootpushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>93</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>shoot</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chattextBrowser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="emceetextBrowser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>451</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="checklineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="checkpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QLabel" name="onvotelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timerlcdNumber">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="shootlcdNumber">
   <property name="geometry">
    <rect>
     <x>673</x>
     <y>242</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
